src/pages/quick-start/commonSteps/createFirebaseVerifier.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Setting up your Firebase Verifier
+---
+
+import VerifierDashboard from "@site/static/images/dashboard/create-verifier.png";
+
+One of the prerequisites of using this SDK, is the creation of a custom verifier. This can be done
```

Comment:
To get started with this SDK, you’ll need to create a custom verifier. Don’t worry—it’s a simple process! You can set it up quickly by enabling custom authentication in your Web3Auth Dashboard. In this guide, we’ll walk you through using Firebase as our Auth Provider to make everything seamless and straightforward.

Can go with easy and inviting sentences, please take a look once